Citizen science (#249)
arcus/education_modules: citizen_science/citizen_science.md

<!--

author:   Rose Hartman
email:    hartmanr1@chop.edu
version:  1.0.0

> module_template_version: 3.1.0

Reviewer: ```suggestion
module_template_version: 3.0.0
```

language: en
narrator: UK English Female
title: Citizen Science
comment:  This is an overview of citizen science for biomedical researchers.
long_description: This module describes what citizen science is, how it is used in research, and what motivates participants to contribute. It also highlights several examples of real citizen science projects being used in biomedical research and related fields. No prior knowledge is assumed.
estimated_time: 30 minutes
@learning_objectives  

After completion of this module, learners will be able to:

- list several ways members of the public can contribute to scientific projects
- identify several different factors that may motivate people to participate in citizen science

> - analyze existing citizen science projects for things like policies on collaboration, goals of the research, and strategies for implementation

Reviewer: This learning objective is definitely focused on in the quiz, but only a little discussed in the module itself. Could some of the content in the quiz be moved to the main portion? Maybe analyze one of the articles "together" and then the other one can be a quiz question.

<div class = "learn-more">
<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>

Read [an interview with biologist Caren Cooper](https://e360.yale.edu/features/interview_caren_cooper_how_rise_of_citizen_science_is_democratizing_research) about her experience with several citizen science projects, including what she's learned about how participating in citizen science can impact individuals and communities.

</div>

Citizen science is attractive to many researchers not only because of the possibility of outsourcing work on large complex data, but because it provides the potential to democratize science and allow the public more influence over and involvement in the scientific process.

<div class = "learn-more">
<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>

Some projects focus especially on how public involvement in science can put power back in the hands of communities.
For example, Public Lab works to pursue environmental justice through community science and open technology. From the [Public Lab About page](https://publiclab.org/about):

> "When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action."

Reviewer: ```suggestion
> When people can easily and reliably track local effects associated with environmental injustices — increased flooding, poor air quality, pollution and destruction of wetlands — they can make better-informed decisions and take action.
```

</div>

The prevalence of citizen science has increased dramatically in recent years, especially as more digital platforms emerge to support it (for example [Zooniverse](https://www.zooniverse.org/), [Globe at Night](https://www.globeatnight.org/), [Foldit](https://fold.it/), and [The Many Brains Project](https://www.testmybrain.org/)).

<div class = "history">
<b style="color: rgb(var(--color-highlight));">Historical context</b><br>

[What Is Citizen Science? – A Scientometric Meta-Analysis](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147152) explores the history of citizen science projects, including which fields make the most use of citizen science and what scientific output they have produced.
It includes a meta-analysis of more than 2000 published papers.

</div>

> ## Why participate in citizen science?

Reviewer: My initial thought was that this section would be about why a scientist would want to structure something as "citizen science" but I'm not sure how to rephrase this to make it clear that this section is about the citizens volunteering.

Author: Ah, good call. I changed it to "What motivates citizen scientists?"

Citizen scientists vary a tremendous amount in terms of the level of dedication they bring to a project, their expertise, and their motivation for participating.
In some cases, participants are attracted just by the prospect of contributing to a scientific project, especially if it's an issue they care about.
They also may be motivated by helping their community or themselves with the output of the research.

> Other times, the project is [designed as a game](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662) or includes more traditional incentives like money to attract participants.

Reviewer: ```suggestion
Other times, the project includes more traditional incentives like money to attract participants, or is designed as a game. Using free smartphone games to collect research data can be quite effective, as this [article in PLOS]((https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100662)  shows.
```

I'm not sure how to make this link both accessible (clear what sort of thing it goes to) and flow nicely, but here is a mediocre suggestion.

## How to build a citizen science project

Large citizen science projects take a substantial effort to get off the ground but are worthwhile because they pay dividends for a long time once they reach a certain momentum.
It may take years before a project produces valuable scientific output.
Successful projects often rely on grant funding to pay for dedicated staff.

For a quicker start, see if there's an existing project you can join rather than starting your own.
If you can contribute time and expertise to an existing project, it may be possible for you to add tests or experiments to their ongoing work.

<div class = "learn-more">
<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>

For an idea of what might be involved in setting up a larger project, see the [Federal Crowdsourcing and Citizen Science Toolkit](https://www.citizenscience.gov/toolkit/#) published by the U.S. government.

</div>

A more reasonable option for researchers looking to get started with citizen science is to post a project on an existing platform, like [Zooniverse](https://www.zooniverse.org/lab).

> ## Quiz: Citizen Science

Reviewer: ```suggestion
## Quiz
```

Reviewer: The template suggests no quiz "topic" if there is only one quiz.

Which of the following are ways citizen scientists can help a project? Check all that apply.

[[X]] data/sample collection
[[X]] data coding
[[X]] scientific programming
[[X]] donating computing power
***
<div class = "answer">

These are all examples of how citizens can support scientific discovery!

</div>
***

Which of the following might motivate someone to participate as a citizen scientist? Check all that apply.

[[X]] learning something interesting
[[X]] wanting to be involved in science
[[X]] payment or gift incentives
[[X]] fun
[[X]] competitiveness
[[X]] wanting to give back to their community
[[X]] wanting support an important cause
***
<div class = "answer">

All of these are reasons people put their time and effort towards citizen science projects!

As you learn about new citizen science projects, notice the language they use on their recruitment materials and think about what kinds of motivation they're attempting to tap into.
If you start your own citizen science project, carefully consider what might best motivate your participants.

</div>
***

> True or False: There are platforms available online to help you launch new citizen science projects quickly.

Reviewer: I was thrown by the word "quickly" given how the module had early emphasized how these projects can take a long time to get momentum to produce scientific results.

Author: Good call! Will revise
```suggestion
True or False: There are platforms available online to help you launch new citizen science projects without having to do all of the setup work yourself.
```

Author: Good call --- I revised

- to do work that would traditionally be done by research staff but doesn't necessarily require any special knowledge or training, especially [collecting measurements or observations](https://malariajournal.biomedcentral.com/articles/10.1186/1475-2875-11-43) or [processing data](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0039808)
- to access [larger and more diverse participant populations](http://cognitivehealth.tech/wp-content/uploads/2019/02/DodellFeder2019-SCZRes.pdf) than would otherwise be available through more traditional recruitment methods

<div class = "learn-more">
<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>

Read [an interview with biologist Caren Cooper](https://e360.yale.edu/features/interview_caren_cooper_how_rise_of_citizen_science_is_democratizing_research) about her experience with several citizen science projects.
She discusses how using citizen science has changed the kinds of research questions she's able to ask, and also the impact participating in science has had on the individuals and communities she works with.

> > In the beginning I didn’t really see the full benefits of citizen science. At first I did have this narrow, selfish view of it, like, “Oh, there are more people to help with the work and that will be great.”...

Reviewer: ```suggestion
> In the beginning I didn’t really see the full benefits of citizen science. At first I did have this narrow, selfish view of it, like, “Oh, there are more people to help with the work and that will be great.” It didn’t take long though to learn two things. One was that I can ask these amazing questions that transcend single study sites. And the other was that I learned that these field experiences and this collaborative relationship between members of the public and scientists actually provide meaningful and really transformative experiences for individuals and for communities. 
```

I like the whole paragraph!
